applications/nvidia_nim/nvidia_nim_imaging/metadata.json

```
@@ -0,0 +1,36 @@
+{
+	"application": {
+		"name": "Imaging using NVIDIA Inference Microservice (NIM)",
+		"authors": [
+			{
+				"name": "Holoscan Team",
+				"affiliation": "NVIDIA"
+			}
+		],
+		"language": "Python",
+		"version": "1.0",
+		"changelog": {
+			"1.0": "Initial Release"
+		},
+		"holoscan_sdk": {
+			"minimum_required_version": "1.0.0",
+			"tested_versions": [
+				"1.0.0"
+			]
+		},
```

Probably `2.1.0` based on `requirements.txt`?

```suggestion
			"minimum_required_version": "2.1.0",
			"tested_versions": [
				"2.1.0"
			]
		},
```

---

I updated `requirements.txt` to use 1.0.3